db/blip_sync_context.go

```diff
@@ -541,8 +541,10 @@ func (bsc *BlipSyncContext) sendNoRev(sender *blip.Sender, docID, revID string,
 // Pushes a revision body to the client
 func (bsc *BlipSyncContext) sendRevision(sender *blip.Sender, docID, revID string, seq SequenceID, knownRevs map[string]bool, maxHistory int, handleChangesResponseDb *Database) error {
 	rev, err := handleChangesResponseDb.GetRev(docID, revID, true, nil)
-	if err != nil {
+	if base.IsDocNotFoundError(err) {
 		return bsc.sendNoRev(sender, docID, revID, seq, err)
+	} else if err != nil {
+		return err
```

Comment:
Is there anything we should add to this error message? What conditions trigger this error message (MessageOverload below)? In this case should we assert that the error message returned has enough information so that we know what is going on or should we log here. In this case, we aren't going to hit the trace logging.

Reply:
I have added additional information to the error returned. A warning gets logged when this returns.